### DISCUSSION TITLE
permission error

### DISCUSSION THREAD

**Discussion by mueller-esch** [dougiteixeira/proxmoxve#361] [Feature request]
Nach update auf Proxmoxve-Integration 3.5.0 wurden erst einmal alle bisher genutzten Entitäten abgeschaltet. Da in der neuen Anleitung auf die zu erneuernden permissions hingewiesen wurde, habe ich die in meiner Proxmox-Konfiguration auch brav eingerichtet. (s. https://github.com/dougiteixeira/proxmoxve#proxmox-permissions) Mit der neu zu konfigurierenden Integration erhielt ich auch folgende Ansicht: 
![Proxmox permission error2](https://github.com/user-attachments/assets/03593ed0-4210-4ffa-8611-5c40c4e4bb43)
Leider sind hier keine Entitäten verknüpft.
Es kam eine weitere Fehlermeldung:
![Proxmox permission error](https://github.com/user-attachments/assets/8ee0508e-d167-4869-852d-fd74f7e45c53)
Frage: kann das damit zusammenhängen, dass ich keine echte Subscription von Proxmox habe?
Wäre für eine zeitnahe Antwort dankbar

**Comment by dougiteixeira** [comment]
This was fixed with #371 in version 3.5.2.

Please, when you have a bug, open an issue instead of a discussion.
